Gems/OpenXRVk/Code/Source/XRCameraMovementComponent.h

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/Component.h>
+#include <AzCore/Component/TickBus.h>
+#include <AzFramework/Input/Events/InputChannelEventListener.h>
+
+#include <OpenXRVk/XRCameraMovementInterface.h>
+
+namespace OpenXRVk
+{
+    /*
```

Comment:
Commenting guidelines dictate using //!

Reply:
Ok I will change it.

Reply:
While I don't think there's anything wrong with the `/**/` comments, I went ahead and changed these to be `//!`.  I looked at https://github.com/o3de/sig-core/blob/main/governance/API-Ref-Guidelines-Update.md to see if it mentioned multi-line comments and it didn't show any in the examples.  However, I instantiated this file from the DefaultComponent template, so I guess either the templates should be updated or multi-line comments are okay to use.

Moving on...